### PR TITLE
DNX Commands: Add --targetProject option

### LINKED ISSUE #2679: DNX Commands: Make ef a global command (enables --starupProject)
Today users need to run `dnx . ef`, it would be better if they could just run a global command on any project without having to do project modification. To do this properly we'd need to have a way to load a runtime environment to harvest information for loading EF things.

/cc @anurse

### PR DESCRIPTION
**Pull request by bricelam**
Resolves #2679 (for now)

### CODE REVIEW

File: src/EntityFramework.Commands/Commands/Program.cs [hunk 1]
```diff
@@ -67,6 +67,9 @@ public static int Main([NotNull] string[] args)
                             var context = update.Option(
                                 "-c|--context <context>",
                                 "The DbContext to use. If omitted, the default DbContext is used");
+                            var targetProject = update.Option(
+                                "-p|--targetProject <project>",
+                                "The project with the Migration classes. If omitted, the current projet is used.");
```
Comment (lajones) [inline src/EntityFramework.Commands/Commands/Program.cs]: Typo: projet -> project. And same in a few more places below.

**Comment by lajones** [comment]
:shipit:

File: src/EntityFramework.Commands/Commands/Program.cs (hunk 1 above)
Comment (bricelam) [inline src/EntityFramework.Commands/Commands/Program.cs]: Doh. :smile:

Comment (davidfowl) [inline src/EntityFramework.Commands/Commands/Program.cs]: Not unixy enough.

`--target-project`

Comment (bricelam) [inline src/EntityFramework.Commands/Commands/Program.cs]: I'll send a PR to kabob-case all the things.

**Pull request closed without merging**